terraform/bin/config.sh

```diff
@@ -18,7 +17,6 @@ source env/$ENVIRONMENT.env
 export TF_IN_AUTOMATION="true"  # https://www.terraform.io/cli/config/environment-variables#tf_in_automation
 export TF_INPUT="false"  # https://www.terraform.io/cli/config/environment-variables#tf_input
 
-export TF_VAR_app_version="$APP_VERSION"
```

Comment:
I think this should still be set here so that we have a version for the "core deployment"